src/main/steps/applicant1/application-submitted/content.ts

```diff
@@ -74,18 +74,24 @@ const en = ({ isDivorce, userCase, partner, referenceNumber, isJointApplication,
     .add(config.get('dates.applicationSubmittedOffsetDays'), 'day')
     .format('D MMMM YYYY')} confirming whether it has been accepted. Check your junk or spam email folder.`,
   line6: `Your ${partner} will then be sent a copy of the application. They will be asked to check the information and respond. If they do not respond then you will be told what you can do next to progress the application.`,
-  line6Solicitor: `Your ${partner}’s solicitor will be contacted by the court, and asked to confirm they are representing them. They will be sent a copy of the application and asked to respond.`,
-  line7: `If you want the application to be served (sent) to your ${partner} by post instead of email, then phone 0300 303 0642.`,
-  subHeading5: 'Dividing your money and property',
-  line8: `It’s usually more straightforward and less expensive if you agree with your ${partner} on how to divide your money and property. <a class="govuk-link" href="https://www.gov.uk/money-property-when-relationship-ends/mediation">Get help agreeing.</a>`,
+  line7: `Your ${partner}’s solicitor will be contacted by the court, and asked to confirm they are representing them. They will be sent a copy of the application and asked to respond.`,
+  line8: `If you want to ‘serve’ (send) the documents to your ${partner} yourself then phone 0300 303 0642 to request it. Otherwise the court will do it.`,
   line9:
+    'If you want the court to serve (send) the application to be served by post instead of by email, then phone 0300 303 0642.',
+  line10: `The address you have provided for your ${partner} is outside of England and Wales. That means you are responsible for ‘serving’ (sending) the court documents, which notify your ${partner} about ${
+    isDivorce ? 'the divorce' : 'ending the civil partnership'
+  }.`,
+  line11: `You will receive the documents that you need to send to your ${partner} by email and letter, after the application has been checked.`,
+  subHeading5: 'Dividing your money and property',
+  line12: `It’s usually more straightforward and less expensive if you agree with your ${partner} on how to divide your money and property. <a class="govuk-link" href="https://www.gov.uk/money-property-when-relationship-ends/mediation">Get help agreeing.</a>`,
```

Comment:
could move this html to the template

Reply:
oh and the links below as well. I don't mind it in the content but just to be consistent with them

Reply:
done